CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -2,7 +2,18 @@
 Opaleites
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
+was mostly covered by marshes, seas and oceans. Their unique resperatory system
+lets them breathe under water as well as on land. An Opaleite's eyes are a 
+little better at seeing underwater then they are out of it.
+
+    Play as an Oplaite if you want to stick close to your team as if they were 
+your schoolmates. If your clan and where you came from is very important to you. 
+If you want to go anywhere in land, sea or air and see what there is to see in 
+this beautiful and diverse universe. Play as an Opalite if what you eat is very
+important to you. Play as a sub-species of Opalite if you want to dazzle the 
+world with your beautiful scales or be the toughest fish in the bay with your 
+bullet-resistant scales. 
```

Comment:
The flow of this par. could use some work: The "If" sentence fragments would work a lot better as a series of clauses in one sen. (a la "schoolmates, if...you, if...universe."). Also, given the context of the doc so far, "Play as an Opalite if what you eat is very important to you" comes out of left field because it's the only mention of diet in the entire doc so far, but it's presented as a critical part of the species.

Reply:
I'll re-write this as best I can, but we never got a design doc for Opaleites. So I don't really know what's supposed to go here, and I don't know if anyone on the team knows.